src/contract.rs

```diff
@@ -124,7 +124,7 @@ pub fn instantiate(
             "contract_info",
             format!("{:?}", get_contract_info(deps.storage)?),
         ),
-        attr("action", "init"),
+        attr("action", ContractAction::Init.to_string()),
```

Comment:
does this change keep the existing casing?

Reply:
ya, i'll double check, but the unit tests do check the result against an expected string. I believe the Serde snake_case property on the enum makes it match the existing case

Reply:
confirmed its snake cased